drivers/webex/instant_connect.cr

```diff
@@ -0,0 +1,37 @@
+class Webex::InstantConnect < PlaceOS::Driver
+  # Discovery Information
+  generic_name :InstantConnect
+  descriptive_name "Webex InstantConnect"
+  uri_base "https://mtg-broker-a.wbx2.com"
+
+  default_settings({
+    bot_access_token: "token",
+  })
+
+  @audience_setting = ""
```

Comment:
missing the extraction of the audience settings
the default setting should be: `a4d886b0-979f-4e2c-a958-3e8c14605e51` (which is CISCOs one and we won't need to change often)

Reply:
Ah ok I wasn't sure if that was secret